src/allotropy/allotrope/models/plate_reader_benchling_2023_09_plate_reader.py

```diff
@@ -1,6 +1,6 @@
 # generated by datamodel-codegen:
 #   filename:  plate-reader.json
-#   timestamp: 2024-03-11T14:24:12+00:00
+#   timestamp: 2024-03-15T12:07:47+00:00
```

Comment:
Can you undo this change, as there do not seem to be any changes in the schema. (there probably were at one point, and then they were reverted, which caused the generation script to update the timestamp, if you revert it it should not change again)